lucien-ios/Dashboard/DashboardTableView/MyCollectionTableViewCell.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  MyCollectionTableViewCell.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/23/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+
+final class MyCollectionTableViewCell: UITableViewCell {
+
+    @IBOutlet private weak var comicImageView: UIImageView!
+    @IBOutlet private weak var comicTitleLabel: UILabel!
+    @IBOutlet private weak var storyTitleLabel: UILabel!
+    @IBOutlet private weak var volumeIssueLabel: UILabel!
+
+    static let reuseIdentifier = "myCollectionCell"
+
+    func configure(comicTitle: String, storyTitle: String, volume: String, issueNumber: String, imageURL: String) {
+        comicTitleLabel.text = comicTitle
+        storyTitleLabel.text = storyTitle
+        volumeIssueLabel.text = "Volume " + volume + ", Issue #" + issueNumber
+        setImage(imageURL: imageURL)
+    }
+
+    // TO DO: load another image created by me if no comic book cover
+    func setImage(imageURL: String?) {
+        guard let imageURL = imageURL else { return }
+        guard let url = URL(string: imageURL) else { return }
+        guard let data = try? Data(contentsOf: url) else {
+            return
+        }
+        comicImageView.image = UIImage(data: data)
+    }
+
+    func setStyling() {
+        comicImageView.clipsToBounds = true
+        comicImageView.layer.cornerRadius = CGFloat(6.0)
+    }
+
+    func styleCell(whiteCellBackground: UIView) {
+        self.contentView.backgroundColor = UIColor.white
```

Comment:
`styleCell` is called repeatedly by the `cellForRowAt` in `DashboardTableView`. The `contentView` of `MyCollectionTableViewCell` contains 4 subviews (its private IBOutlets). One way to prevent this issue is to check if the number of subviews is equal to 4. That way you can only add a shadow to brand new cells.

`if contentView.subviews.count == 4  { ...`

Reply:
That's not a good idea - anything that involves relying on UI (e.g. number of subviews) is a code smell. Your code should never rely on UI for functionality. I agree that you should only add a shadow once, however the way to do that is to store the subview inside the `UITableViewCell` as an optional property and add it if it's nil